app/views/site/faq.scala

```diff
@@ -357,63 +357,48 @@ object faq:
         ),
         question(
           "make-a-bot",
-          "Make a Lichess bot?",
+          makeABotQ.txt(),
           p(
-            "To learn how to create a ",
-            a(href := "https://lichess.org/blog/WvDNticAAMu_mHKP/welcome-lichess-bots")("Lichess bot"),
-            ", please read ",
-            a(href := "https://lichess.org/@/thibault/blog/how-to-create-a-lichess-bot/FuKyvDuB")(
-              "this blog post"
-            ),
-            "."
+            makeABotA(
+            a(href := "https://lichess.org/blog/WvDNticAAMu_mHKP/welcome-lichess-bots")("Lichess Bot"),
+            a(href := "https://lichess.org/@/thibault/blog/how-to-create-a-lichess-bot/FuKyvDuB")(thisBlogPost())
+            )
           )
         ),
         question(
           "autoplay",
-          "Enable autoplay for sounds?",
-          p("""
-            Most browsers can prevent sound from playing on a freshly loaded page to protect users. 
-            Imagine if every website could immediately bombard you with audio ads.
-          """),
-          p("""
-            The red mute icon appears when your browser prevents lichess.org from playing a sound.
-            Usually this restriction is lifted once you click something. On some mobile browsers,
-            touch dragging a piece does not count as a click. In that case you must tap the board to
-            allow sound at the start of each game.
-          """),
-          p("""
-            We show the red icon to alert you when this happens. Often you can explicitly allow
-            lichess.org to play sounds. Here are instructions for doing so on recent versions of
-            some popular browsers.
-          """),
-          h3("Mozilla Firefox (desktop):"),
+          enableAutoplayForSoundsQ.txt(),
+          p(
+            enableAutoplayForSoundsA()
+          ),
+          h3(browsersMozillaFirefox()),
           ul(
-            li("Go to lichess.org"),
-            li("Press ctrl-i on linux/windows or cmd-i on macos"),
-            li("Click the Permissions tab"),
-            li("Allow Audio and Video on lichess.org")
+            li(goToLichess()),
```

Comment:
These 4 `ul`s should each be converted to a `p` that contains only one translation. That way we'll need 4 times fewer translation keys.

Reply:
I mean putting all the `li` texts into a single `p`.

Reply:
@M-DinhHoangViet If you setup a development site using those instructions, you can test it and see if it works.